examples/util/base.js

```diff
@@ -15,7 +15,8 @@ const preferMap = {
   'low': 'LOW_POWER',
 };
 
-const mobilenet_v1_tflite = {
+
+const imageClassificationModels = [{
```

Comment:
@ibelem Such variables **mobilenet_v1_tflite**,  **mobilenet_v2_tflite** were invoked in index.js#L24 of Benchmark Test, this modification would produce ReferenceError of Benchmark Test. Or you could merge this code, I will update Benchmark Test with this modification.

Reply:
I prefer to merge this PR at first.